taxi_service/settings.py

```diff
@@ -44,6 +44,8 @@
     "django.contrib.staticfiles",
     "debug_toolbar",
     "taxi",
+    "crispy_forms",
+    "crispy_bootstrap4",
```

Comment:
The inclusion of `crispy_forms` and `crispy_bootstrap4` in `INSTALLED_APPS` is correct. Ensure that all forms in the project are utilizing crispy forms as intended.